app.py

```diff
@@ -1,12 +1,47 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
+import pprint
+pp = pprint.PrettyPrinter(indent=4)
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 @app.route('/')
-def index():
-    return 'Hello World!'
-
+def index(): 
+    items = session.get_items()
+    return render_template('index.html' , items=items)
 if __name__ == '__main__':
```

Comment:
I'd put these 2 lines at the bottom of the program (in fact you don't _really_ need them at all)